packages/main/src/ColorPalette.js

```diff
@@ -0,0 +1,184 @@
+import ColorPaletteEntry from "./ColorPaletteEntry.js";
+import {
+	COLORPALETTE_CONTAINER_LABEL,
+} from "./generated/i18n/i18n-defaults.js";
+
+// Styles
+import ColorPaletteCss from "./generated/themes/ColorPalette.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-color-palette",
+	managedSlots: true,
+	properties: /** @lends sap.ui.webcomponents.main.ColorPalette.prototype */ {
+		/**
+		 *
+		 * The selected color.
+		 * @type {CSSColor}
+		 * @public
+		 */
+		value: {
+			type: CSSColor,
+		 },
+		/**
+		 * The colors grouped in object with their indexes.
+		 * @type {CSSColor}
+		 * @private
+		 */
+		entries: {
+			type: Object,
+			multiple: true,
+		 },
+	},
+	slots: /** @lends sap.ui.webcomponents.main.ColorPalette.prototype */ {
+		/**
+		 * Defines the <code>ui5-color-palette-entry</code> items.
+		 * @type {HTMLElement[]}
+		 * @slot
+		 * @public
+		 */
+		"default": {
+			propertyName: "colors",
+			type: HTMLElement,
+			invalidateOnChildChange: true,
+			individualSlots: true,
+		},
+	},
+	events: /** @lends sap.ui.webcomponents.main.ColorPalette.prototype */ {
+		/**
+		 * Fired when the user selects a color.
+		 *
+		 * @event
+		 * @public
+		 * @param details
+		 */
+		change: {
+			details: {
+				color: {
+					type: "CSSColor",
+				},
+			},
+		 },
+	},
+};
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ * The ColorPalette provides the users with a range of predefined colors.
+ * You can set them by using the ColorPaletteEntry items as slots.
+ *
+ * <h3>Usage</h3>
+ * The palette is intended for users, who don't want to check and remember the different values of the colors .
+ *
+ * For the <code>ui5-color-palette</code>
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import @ui5/webcomponents/dist/ColorPalette.js";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.main.ColorPalette
+ * @extends UI5Element
+ * @tagname ui5-color-palette
+ * @since 1.0.0-rc.12
+ * @appenddocs ColorPaletteEntry
+ * @public
+ */
+class ColorPalette extends UI5Element {
+	static get metadata() {
+		return metadata;
+	}
+
+	static get render() {
+		return litRender;
+	}
+
+	static get styles() {
+		return ColorPaletteCss;
+	}
+
+	static get template() {
+		return ColorPaletteTemplate;
+	}
+
+	static get dependencies() {
+		return [ColorPaletteEntry];
+	}
+
+	static async onDefine() {
+		await fetchI18nBundle("@ui5/webcomponents");
+	}
+
+	constructor() {
+		super();
+		this.i18nBundle = getI18nBundle("@ui5/webcomponents");
+		this._itemNavigation = new ItemNavigation(this, {
+			getItemsCallback: () => this.colors.slice(0, 15),
+			rowSize: 5,
+			behavior: ItemNavigationBehavior.Cyclic,
+		});
+	}
+
+	onBeforeRendering() {
+		if (!this.entries.length) {
+			this.colors.forEach((item, index) => {
+				item.index = index + 1;
+			});
+		}
+	}
+
+	selectColor(target) {
+		target.getDomRef().focus();
```

Comment:
`target.focus()` is better, because it does: 

```js
async focus() {
		await this._waitForDomRef();

		const focusDomRef = this.getFocusDomRef();

		if (focusDomRef && typeof focusDomRef.focus === "function") {
			focusDomRef.focus();
		}
	}
```

Reply:
Done locally will be pushed when we clarify the naming.